typescript/web/src/pages/posts/[slug].tsx

```diff
@@ -36,7 +36,7 @@ export default function Posts({
           article?.image?.url !== null
             ? [
                 {
-                  url: article.image.url,
+                  url: article?.image?.url,
```

Comment:
Won't it be safer to replace `article?.image?.url !== null` by just `article?.image?.url` instead of ignoring `undefined` by using `?` in line 39 (` url: article?.image?.url,`)?

Reply:
That's correct, let's do that in another PR, I merged right after @Jordanlelay approved

Reply:
https://github.com/labelflow/labelflow/issues/646